web/files/files_test.go

```diff
@@ -1025,7 +1025,10 @@ func TestFiles(t *testing.T) {
             "tags": ["bar", "bar", "baz"],
             "name": "moved",
             "dir_id": "` + dirID + `",
-            "executable": true
+            "executable": true,
+			"cozyMetadata": {
+			  "favorite": true
+			}
```

Comment:
The indent looks weird

Reply:
Good catch, there is a mix of spaces and tabs